src/shared/components/plots/ScatterPlotUtils.ts

```diff
@@ -9,3 +9,45 @@ export function getRegressionComputations(
 } {
     return regression.linear(data);
 }
+
+export const makeMultilineAxisLabel = (
```

Comment:
Please follow the function declaration syntax we use elsewhere. See line 3 of this file.